octree_web_viewer/client/index.html

```diff
@@ -14,17 +14,19 @@
   limitations under the License.
 -->
 <!doctype html>
+
 <head>
 </head>
+
 <body>
   <div id="renderArea" style="position: absolute; width: 100%; height: 100%; overflow: hidden;"></div>
 
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.5.1/dat.gui.min.js"></script>
-  <script src="https://cdnjs.cloudflare.com/ajax/libs/three.js/r82/three.min.js"></script>
-  <script data-main="app_bundle" src="https://cdnjs.cloudflare.com/ajax/libs/require.js/2.2.0/require.min.js"></script>
+  <!--script src="https://cdnjs.cloudflare.com/ajax/libs/dat-gui/0.7.2/dat.gui.min.js"></script>
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/three.js/0.101.1/three.min.js"></script>
+  -->
```

Comment:
Can you please completely delete the lines instead of commenting them out?

Reply:
done!